package.json

```diff
@@ -1,31 +1,71 @@
 {
-    "name": "scripture-burrito-store",
-    "version": "0.0.0",
-    "description": "",
-    "license": "Apache-2.0",
-    "main": "run_burrito.js",
-    "dependencies": {
-        "ajv": "^6.10.2",
-        "chai": "^4.2.0",
-        "crypto-js": "^4.0.0",
-        "deep-equal": "^2.0.1",
-        "eslint-formatter-pretty": "^3.0.1",
-        "esm": "^3.2.25",
-        "fs-extra": "^8.1.0",
-        "md5-file": "^4.0.0",
-        "minimalist": "^1.0.0",
-        "prettier": "^1.19.1",
-        "pretty-format": "^25.2.6",
-        "readdirp": "^3.4.0",
-        "semver": "^6.3.0",
-        "sha3": "^2.1.1",
-        "uuid": "^7.0.3",
-        "xmldom": "^0.3.0"
-    },
-    "devDependencies": {
-        "eslint": "^6.8.0",
-        "eslint-config-airbnb-base": "^14.1.0",
-        "eslint-plugin-import": "^2.20.2",
-        "eslint-plugin-mocha": "^6.3.0"
+  "name": "scripture-burrito-store",
+  "version": "0.0.0",
+  "description": "",
+  "license": "Apache-2.0",
+  "main": "run_burrito.js",
+  "scripts": {
+    "start": "node express/bin/www",
+    "express": "node express/bin/www",
+    "test": "mocha test/test_all.js",
+    "testAll": "mocha test/test_all.js",
+    "testSmoke": "mocha test/test_smoke.js",
+    "testSchema": "mocha test/test_schema.js",
+    "testDbl": "mocha test/test_dbl_import.js",
+    "testBundleImport": "mocha test/test_bundle_import.js",
+    "testDblOab": "mocha test/test_dbl_oab_import.js",
+    "testSB01": "mocha test/test_sb01_import.js"
+  },
+  "dependencies": {
+    "adm-zip": "^0.4.14",
+    "ajv": "^6.10.2",
+    "babel-core": "^6.26.3",
+    "chai": "^4.2.0",
+    "cookie-parser": "^1.4.5",
+    "crypto-js": "^4.0.0",
+    "deep-equal": "^2.0.1",
+    "deepcopy": "^2.0.0",
+    "esm": "^3.2.25",
+    "express": "^4.17.1",
+    "express-fileupload": "^1.1.7-alpha.3",
+    "fs": "0.0.1-security",
+    "fs-extra": "^8.1.0",
+    "hbs": "^4.1.1",
+    "http-error": "0.0.6",
+    "http-errors": "^1.7.3",
+    "md5-file": "^4.0.0",
+    "minimalist": "^1.0.0",
+    "morgan": "^1.10.0",
+    "optimist": "^0.6.1",
+    "readdirp": "^3.4.0",
+    "semver": "^6.3.0",
+    "sha3": "^2.1.1",
+    "uuid": "^7.0.3",
+    "xmldom": "^0.3.0"
+  },
+  "devDependencies": {
+    "debug": "^4.1.1",
+    "eslint": "^6.8.0",
+    "eslint-config-airbnb-base": "^14.1.0",
+    "eslint-plugin-import": "^2.20.2",
+    "eslint-plugin-mocha": "^6.3.0",
+    "eslint-formatter-pretty": "^3.0.1",
+    "prettier": "^1.19.1",
+    "pretty-format": "^25.3.0",
+    "tape": "^4.13.2",
+    "mocha": "^7.1.1"
+  },
+  "eslintConfig": {
```

Comment:
@smorrison maybe this eslintConfig info should move to test/eslintrc.json since it should only apply to tests?